### PR TITLE
del: add German translation

### PR DESCRIPTION
**Pull request by marciska**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.ca/windows/del.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# del
```
Comment (marchersimon) [inline pages.ca/windows/del.md:1]: I think you might have picked the wrong directory here.

File: pages.ca/windows/del.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Fordere vor dem Löschen jeder Datei zur Bestätigung auf:
+
+`del {{datei_muster}} /p`
+
+- Erzwinge das Löschen von schreibgeschützten Dateien:
+
+`del {{datei_muster}} /f`
+
+- Lösche rekursiv alle Dateien die dem Muster entsprechen in allen Unterordnern:
```
Comment (marchersimon) [inline pages.ca/windows/del.md:18]: How about this?
```suggestion
- Lösche alle Dateien die dem Muster entsprechen rekursiv in allen Unterordnern:
```

File: pages.ca/windows/del.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`del {{datei_muster}} /s`
+
+- Keine Eingabeaufforderung wenn Dateien basierend auf einem globalen Platzhalter gelöscht werden sollen:
```
Comment (marchersimon) [inline pages.ca/windows/del.md:22]: Descriptions should always be in imperative mood.
```suggestion
- Zeige keine Eingabeaufforderung wenn Dateien basierend auf einem globalen Platzhalter gelöscht werden sollen:
```

File: pages.ca/windows/del.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`del {{datei_muster}} /q`
+
+- Hilfe anzeigen und verfügbare Attribute auflisten:
```
Comment (marchersimon) [inline pages.ca/windows/del.md:26]:
```suggestion
- Zeige Hilfe an und liste verfügbare Attribute auf:
```

File: pages.ca/windows/del.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`del /?`
+
+- Lösche Dateien mit gegebenen Attributen:
```
Comment (marchersimon) [inline pages.ca/windows/del.md:30]:
```suggestion
- Lösche Dateien mit den gegebenen Attributen:
```

File: pages.ca/windows/del.md (hunk 1 above)
Comment (marciska) [reply pages.ca/windows/del.md]: Correct. Didn't notice. Probably mis-clicked on the website in the table 😅 Thanks!

**Comment by marciska** [comment]
I agree with the changes. Also, the file is now in the correct translation folder.

File: pages.de/windows/del.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+# del
+
+> Lösche eine oder mehrere Dateien.
+> Mehr Informationen: <https://learn.microsoft.com/windows-server/administration/windows-commands/del>.
+
+- Lösche eine oder mehrere durch Leerzeichen getrennte Dateien oder Datei-Muster (Datei-Namen mit Platzhaltern, "Wildcards"):
+
+`del {{datei_muster}}`
```
Comment (marchersimon) [inline pages.de/windows/del.md:8]: `Dateimuster` is one word, so it shouldn't have an underscore in it.

**Comment by marciska** [comment]
Changed the name as requested.

File: pages.de/windows/del.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# del
+
+> Lösche eine oder mehrere Dateien.
+> Mehr Informationen: <https://learn.microsoft.com/windows-server/administration/windows-commands/del>.
```
Comment (marchersimon) [inline pages.de/windows/del.md:4]:
```suggestion
> Weitere Informationen: <https://learn.microsoft.com/windows-server/administration/windows-commands/del>.
```

**Review by marchersimon — APPROVED** [review]
Thanks @marciska

File: pages.de/windows/del.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+# del
+
+> Lösche eine oder mehrere Dateien.
+> Weitere Informationen: <https://learn.microsoft.com/windows-server/administration/windows-commands/del>.
+
+- Lösche eine oder mehrere durch Leerzeichen getrennte Dateien oder Dateimuster (Dateinamen mit Platzhaltern, "Wildcards"):
```
Comment (kbdharun) [inline pages.de/windows/del.md:6]:
```suggestion
- Lösche eine oder mehrere durch Leerzeichen getrennte Dateien oder Dateimuster:
```
Is this required?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**